composeApp/src/iosMain/kotlin/com/daedan/festabook/presentation/placeMap/platform/NaverMap.ios.kt

```
@@ -0,0 +1,70 @@
+
+    actual fun setOnMapClickListener(onClick: (LatLng) -> Unit) {
+    }
+
+    actual fun setContentPadding(
+        left: Int,
+        top: Int,
+        right: Int,
+        bottom: Int,
+        animate: Boolean,
+    ) {
+    }
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)